src/predict.Rmd

```diff
@@ -133,6 +133,32 @@ As mentioned above, the predicted posterior probabilities can be accessed via th
 head(getPredictionProbabilities(pred))
 ```
 
+### Extract Standard Errors
+
+Some Learners provide standard errors for predictions, which can be accessed in mlr. An overview is given by calling the function[&listLearners]. In order to include learners from packages which are not installed in the overview, one has to set ``check.packages``  as ``FALSE``.
+
+```{r, results='hide', warning=FALSE}
+listLearners("regr", check.packages = FALSE)
+```
+
+In this example we train a Gaussian Process Regression on the Boston Housing dataset. Since the Gaussian Process, ``regr.km``, does not allow factor data, we have to use only numerical variables  for the training. For the learner, the ``predict.type`` has to be specified as correctly:
```

Comment:
Honestly, I'm never totally sure, what is best. :)

A suggestion we got a few times regarding the tutorial was that it would be more beginner-friendly if examples, particularly in Basics, would just focus on the functionality which is explained at that time (here, SE estimation) and to leave out anything that could potentially trip people (including learners that they might not know).
I don't know if GBs really could bother people, in my opinion it's a standard method, too, but LMs are just safer in this regard.